### ISSUE TITLE
Text Align does not work properly when TextArea.text is empty

### ISSUE THREAD

**Issue by weixiongmei** [lvgl/lvgl#1603]
**Describe the bug**

When the TextArea.text is empty(length = 0), the cursor does not align what's set by lv_textarea_set_text_align, it always aligning to the left. LVGL7.0.2

**To Reproduce**
Because the text is set to "", which is zero in length, the cursor ignores the text align setting, the cursor will always aligning to the left.
```C
lv_obj_t * textarea = lv_textarea_create(lv_scr_act(), NULL);
lv_textarea_set_one_line(textarea, true);
lv_textarea_set_text(textarea, ""); 
lv_textarea_set_text_align(textarea, LV_LABEL_ALIGN_CENTER); 
```

When the text is not empty, the cursor working as expected.
```C
lv_obj_t * textarea = lv_textarea_create(lv_scr_act(), NULL);
lv_textarea_set_one_line(textarea, true);
lv_textarea_set_text(textarea, "hello"); 
lv_textarea_set_text_align(textarea, LV_LABEL_ALIGN_CENTER); 
```

**Screenshots**
![IMG_1030 2](https://user-images.githubusercontent.com/15250619/85465692-1669f580-b577-11ea-9571-b6b40f70e933.jpg)
In the picture you can see, the first text area is empty, and it is set to align_center, the placeholder is centered, but the cursor is not. The second text area is set to align_center as well, the only different is the text is not empty, so it's working as expected.

**Comment by kisvegabor** [comment]
Please fill the template to provide more information. A screenshot would be really helpful in this case.
```
> # Important: issues that don't use this template will be ignored/closed.

**Describe the bug**

A clear and concise description of what the bug is.

**To Reproduce**

Please provide a small, independent code sample that can be used to reproduce the issue. Ideally this should work in the PC simulator unless the problem is specific to one platform.

**Expected behavior**

A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**

Add any other context about the problem here.

```

**Comment by weixiongmei** [comment]
Information have been updated, please help~~ Thanks

**Comment by kisvegabor** [comment]
Thanks for the update.
I fixed it here: 63aba1b3

**Comment by kisvegabor** [comment]
As the issue seems to be solved I close it.

If it's still not working please reopen the issue and comment here.
